packages/node/src/indexer/x-provider/http.ts

```diff
@@ -54,7 +54,7 @@ export class HttpProvider implements ProviderInterface {
 
   readonly #stats: ProviderStats;
 
-  readonly #ctx: ReturnType<typeof context>;
+  readonly ctx: ReturnType<typeof context>;
```

Comment:
Whats the reason for this change? It removes the privacy of the variable

Reply:
```
Cannot read private member #ctx from an object whose class did not declare it
TypeError: Cannot read private member #ctx from an object whose class did not declare it
    at Proxy.disconnect (/Users/jiatwork/ofn/subql/packages/node/src/indexer/x-provider/http.ts:122:16)
    at RpcCore.disconnect (/Users/jiatwork/ofn/subql/node_modules/@polkadot/rpc-core/cjs/bundle.js:103:30)

```

It throwing this error, I feel it should be safe to remove # here as it only accessed from polkadot